src/templates/bootstrap5/builderWizard/form.ejs

```diff
@@ -3,13 +3,13 @@
     {{ctx.sidebar}}
   </div>
   <div class="col-xs-8 col-sm-9 col-md-10 formarea">
-    <ol class="breadcrumb">
```

Comment:
Please also add this change to the @formio/bootstrap repo since 5.x exclusively uses that repo.